airbyte-integrations/connectors/source-mailchimp/source_mailchimp/client.py

```diff
@@ -63,19 +63,25 @@ def lists(self, state: DefaultDict[str, any] = None) -> Generator[AirbyteMessage
         stream_name = self._LISTS
         date_created = self._get_cursor_or_none(state, stream_name, cursor_field)
 
-        default_params = {"since_date_created": date_created, "sort_field": cursor_field, "sort_dir": "ASC"}
+        default_params = {"sort_field": cursor_field, "sort_dir": "ASC"}
```

Comment:
can you put lines 66 through 71 in a function and reuse it for campaigns?

Reply:
don't need to do this now if you're still iterating and figuring stuff out.

Reply:
DRY'd it into a helper...unclear if it's more readable? it's certainly less repeated code.